lib/rules/no-deprecated-api.js

```diff
@@ -745,6 +746,8 @@ module.exports = {
         messages: {
             deprecated:
                 "{{name}} was deprecated since v{{version}}{{replace}}.",
+            removed:
+                "{{name}} was deprecated since v{{version}}, and removed in v{{removed}}.",
```

Comment:
It could be worth including `replace` here still

Reply:
yes, we can add it in another PR.